tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -610,7 +615,7 @@ Function .onInstSuccess
     ${GetOptions} $CMDLINE "/R" $R0
     ${IfNot} ${Errors}
       ${GetOptions} $CMDLINE "/ARGS" $R0
-      Exec '"$INSTDIR\${MAINBINARYNAME}.exe" $R0'
+      nsis_tauri_utils::RunAsUser "$INSTDIR\${MAINBINARYNAME}.exe" "$R0"
```

Comment:
```suggestion
      Call RunMainBinary
```

Reply:
I don't want to use `Push $R0` to be honest

Reply:
I see, there is a hacky way to avoid it by always using $R9 as the second argument in the function which should be empty for GUI and when checking `/R`, instead of reading into $R0, we read into $R9, but that's not an ideal solution and $R9 could be used later so let's keep it this way for now or convert it into a macro later